app/assets/javascripts/datatables.js

```diff
@@ -1,8 +1,9 @@
 $( document ).on("turbolinks:load", function() {
   $('#passengers').DataTable({
     fixedHeader: true,
+    lengthMenu: [ [10, 25, 50, -1], [10, 25, 50, 'All'] ],
+    order: [[0, 'asc']],
```

Comment:
This isn't strictly related, but if the view starts out ordered (it does), the UI should reflect that.